test/unit/test_p256.ml

```diff
@@ -0,0 +1,231 @@
+module Testable = struct
+  let ok_or_error =
+    let pp ppf = function
+      | Ok () -> Format.fprintf ppf "ok"
+      | Error e -> Format.fprintf ppf "%a" Fiat_p256.pp_error e
+    in
+    let eq ra rb =
+      match (ra, rb) with
+      | Ok (), Ok () -> true
```

Comment:
Also I just realized I forgot to mention that `alcotest` already has a `result` testable combinator which you could use here!

Reply:
Ah, right. This evolved from a piece of code where I couldn't use this. Fixed!